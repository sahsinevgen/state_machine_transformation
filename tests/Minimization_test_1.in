4
3
2
2 4

8
1 2 a
2 1 a
3 4 a
4 3 a
1 4 b
4 1 b
2 3 b
3 2 b