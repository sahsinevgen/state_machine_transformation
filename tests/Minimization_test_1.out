2
1
1
2 

4
1 2 a
1 2 b
2 1 a
2 1 b